src/main/resources/db/changelog/changes/17-create-order-item-table.yaml

```diff
@@ -0,0 +1,42 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-order-item-table
+      author: Mykyta Zuber
+      changes:
+        - createTable:
+            tableName: order_item
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: orders_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+                    foreignKeyName: orders_id_fk

```

Comment:
and to which table and column it references? specify param `references` for all foreign keys